### DISCUSSION TITLE
Question about competitive inhibition

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#878] [Questions & Problems] [answered]
Dear all,

In PK-Sim, when creating a competitive inhibition, in the description box, it is reported the following (this screenshot is taken from the itraconazole model)

![image](https://user-images.githubusercontent.com/11489621/110663076-6c92cd80-81c6-11eb-9604-cc495bc92496.png)

As you can see, in the description it is mentioned that inhibition "Can be overcome by high substrate concentrations". But on the other side, when looking at the equations of the competitive inhibition, the substrate concentrations are not taken into account. As you can see in the formula below, the inhibitor only affects the clearance of the victim drug, but the victim concentrations are not taken into account (this is in agreement with all the formulas I have seen for DDI, so I think the implemented equations are correct).

![image](https://user-images.githubusercontent.com/11489621/110663378-a7950100-81c6-11eb-9340-728a7aff6a16.png)

But then, could you please help me to understand the specification "Can be overcome by high substrate concentrations", since I am trying to better understand the interaction as it is described in PK-Sim.

Thanks a lot in advance for your help!

Donato

**Comment by StephanSchaller** [comment, marked as answer]
Hi @teutonicod, the substrate concentrations are "**S**".

With the competitive inhibition, only the Km is increased (no effect on Vmax). This means, if your victim (i.e., substrate) concentrations are always >> Km (also >> Km,app in the inhibited case), the inhibition will not have a significant effect as the metabolic rate is running at Vmax in either case.

Let me know if this was understandable.

Cheers, Stephan

**Comment by StephanSchaller** [comment]
Maybe "can be overcome" is not an ideal wording to paraphrase this :-)

**Comment by teutonicod** [comment]
Hi @StephanSchaller,

Thanks for your comment. Indeed it supports my impression that the wording "Can be overcome by high substrate concentrations" is not really correct. Let's hear also other opinion, but I think we should consider if changing the description if this does not reflect reality.

Anyway I was very interested in what you mention. If I understood your point, if the victim has concentrations >> Km, the metabolic rate will run at Vmax and the inhibitor should not have anymore an impact. I agree!
But I am just wondering what happens in the frequent case that you have just a linear clearance that is assigned to a specific enzyme. I checked the equations, and in this case, the inhibition factor on the slide is applied to the linear clearance instead of Km. In this case, the clearance will always be affected by the inhibitor, right? 

Best,

Donato
